package.json

```diff
@@ -87,5 +87,6 @@
     "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.1.10",
     "webpack-merge": "^4.1.4"
-  }
+  },
+  "proxy": "http://localhost:4000/"
```

Comment:
why are you adding this?

Reply:
oh, forgot we are no longer using `react auto generated app`. i added to establish connection to backend. but i will remove it on next commit